app/assets/javascripts/browsered/web-payments/index.js

```diff
@@ -9,9 +9,16 @@ const { createGooglePaymentRequest, googlePayNow } = require('./google-pay')
 const paymentMethodForm = document.getElementById('web-payments-container')
 const standardMethodContainer = document.getElementById('enter-card-details-container')
 
+const titleCase = function stringToTitleCase(string) {
+  return string.replace(/\w\S*/g, txt => {
+    return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase()
+  })
```

Comment:
what does this function do?

Reply:
the form submission has values like `apple-payz so it make it into `Apple Pay` which means that all our google analytics categories are matching